src/link.c

```diff
@@ -1233,6 +1365,7 @@ ASYNC_OPERATION_HANDLE link_transfer_async(LINK_HANDLE link, message_format mess
                             DELIVERY_INSTANCE* pending_delivery = GET_ASYNC_OPERATION_CONTEXT(DELIVERY_INSTANCE, result);
                             if (pending_delivery == NULL)
                             {
+                                LogError("Failed creating pending delivery");
```

Comment:
>creating [](start = 49, length = 8)

Is this a create?  Call looks more like a GET

Reply:
Thanks, it is a get. Changed.

---
In reply to: [173501836](https://github.com/Azure/azure-uamqp-c/pull/223#discussion_r173501836) [](ancestors = 173501836)